incuna-sass/mixins/_font-smoothing.sass

```diff
@@ -0,0 +1,27 @@
+// Font smoothing mixin to add / remove antialising -moz- and -webkit- 
+// font smoothing properties to text
+
+// It's important to note that font-smoothing is not recommended for all 
+// situations and is not supported by all browsers
+// see: http://usabilitypost.com/2012/11/05/stop-fixing-font-smoothing/
+
+// Using font-smoothing antialised can be useful if, for example, you're 
+// using white font on a dark background and the browser is rendering it
+// too bold
+
+// To set font-smoothing to antialiased 
+//     .thing
+//         @include font-smoothing
+// antialised is the default setting so no arguments are necessary
+
+// To set font-smoothing to auto, or overwrite antialised
+//     .thing
+//         @include font-smoothing(auto)
+
+@mixin font-smoothing($type: antialiased)
+    @if $type == antialiased
+        -moz-osx-font-smoothing: grayscale
+        -webkit-font-smoothing: antialiased
+    @else 
```

Comment:
Related to above, can you warn the user if they're passing an unknown argument?
```sass
@mixin font-smoothing($type: antialiased)
    @if $type == antialiased
        -moz-osx-font-smoothing: grayscale
        -webkit-font-smoothing: antialiased
    @else if $type == auto
        -moz-osx-font-smoothing: auto
        -webkit-font-smoothing: auto
    @else
        @warn "Unknown argument: #{$type}"
```

Then
```sass
.thing
    @include font-smoothing(crazy)
```
outputs
```css
WARNING: Unknown argument: crazy
Backtrace:
	test.sass:9, in mixin `font-smoothing`
	test.sass:12
```